packages/core-app-api/config.d.ts

```diff
@@ -25,6 +25,12 @@ export interface Config {
      */
     baseUrl: string;
 
+    /**
+     * The URL to redirect to upon sign out.
+     * @visibility frontend
+     */
+    signOutUrl?: string;
```

Comment:
I would recommend creating a `redirect_path` configuration  under a `logout` key. It prepared us to create other logout related configs in the future like a time to auto logout users for example. So in short:
```ts
logout: {
  redirect_path?: string;
}
```
Accepting a path instead of a url might be less risky than allowing redirection to external addresses.

Reply:
Having a dedicated section for these options is a great idea, as this functionality is likely to have other related configuration options. I am indifferent to what this section is named, though I did use `signOut` as this is the current nomenclature used by Backstage.

As for `redirect_path`, I believe that URL is the appropriate attribute type given that it is very likely that an external URL will be necessary. This is outlined in my long-form description of the use case above.

Reply:
Got it. Maybe updating the config to something like `{ logout: { redirectUrl?: string } }` would be an alternative, but let's hear what @Rugvip thinks too, it could not be worth it 🙂.

Reply:
I'm open to that as well. I agree on consulting @Rugvip.

Thanks again!

Reply:
As discussed in the framework SIG we'll want to aim to keep as much of the logout logic in the auth providers in the backend. Anything that can be implemented in the authenticator `logout` should be there.

We'll also want try to to hook up this logout URL to the logout endpoint. I'm thinking we'll be able to return an optional logout URL from the provider, which potentially lets us generate the URL based on information that the provider has. That should also let us handle the case of multiple auth providers and environments that need different logout URLs.